Gulpfile.js

```diff
@@ -126,7 +126,18 @@ gulp.task('clean', function (cb) {
 
 
 // Lint
-gulp.task('lint', function () {
+gulp.task('lint-client', function () {
```

Comment:
I'd prefer to move client scripts linting to the separate task now and join it with the main `lint` task after the refactoring